web/src/main/java/com/microsoft/applicationinsights/web/internal/config/WebReflectionUtils.java

```diff
@@ -1,21 +1,23 @@
-package com.microsoft.applicationinsights.internal.config;

```

Comment:
If we're releasing 2.5, I don't think we can move this class since it's public. Unless there's a technical reason why it needs to be in this package.

Reply:
this is a good question, the reason for the move may not be applicable anymore, i'll look into it